networkit/cpp/centrality/test/CentralityGTest.cpp

```diff
@@ -774,6 +777,125 @@ TEST_F(CentralityGTest, testApproxClosenessCentralityOnToyGraph) {
     EXPECT_NEAR(0.2, maximum2, tol);
 }
 
+TEST_F(CentralityGTest, testApproxClosenessCentralityOnDirectedToyGraph) {
+    /* Graph:
+     0 -> 3
+        /
+       2    5
+      /  \ /
+     1    4
+    */
+    count n = 6;
+    Graph G(n, false, true);
+
+    G.addEdge(0, 3);
+    G.addEdge(2, 0);
+    G.addEdge(1, 2);
+    G.addEdge(2, 1);
+    G.addEdge(3, 2);
+    G.addEdge(2, 3);
+    G.addEdge(4, 2);
+    G.addEdge(2, 4);
+    G.addEdge(4, 5);
+    G.addEdge(5, 4);
+
+    ApproxCloseness acc(G, 6, 0.1, true);
+    acc.run();
+    std::vector<double> cc = acc.scores();
+    double tol = 0.01;
+
+    EXPECT_NEAR(acc.maximum(), 0.2, tol);
+    EXPECT_NEAR(0.38, cc[0], tol);
+    EXPECT_NEAR(0.5, cc[1], tol);
+    EXPECT_NEAR(0.833335, cc[2], tol);
+    EXPECT_NEAR(0.5, cc[3], tol);
+    EXPECT_NEAR(0.625, cc[4], tol);
+    EXPECT_NEAR(0.417, cc[5], tol);
+}
+
+TEST_F(CentralityGTest, testBetweennessMaximum) {
+    /* Graph:
+     0    3
+      \  / \
+       2    5
+      /  \ /
+     1    4
+    */
+    count n = 6;
+    Graph G(n);
+
+    G.addEdge(0, 2);
+    G.addEdge(1, 2);
+    G.addEdge(2, 3);
+    G.addEdge(2, 4);
+    G.addEdge(3, 5);
+    G.addEdge(4, 5);
+
+    Betweenness bet(G, false, false);
+    bet.run();
+    EXPECT_EQ(10, bet.maximum());
+    EXPECT_NEAR(1.79, bet.centralization(), 0.01);
+
+    G.indexEdges();
+    Betweenness betNormalized(G, true, true);
+    betNormalized.run();
+    EXPECT_EQ(1, betNormalized.maximum());
+    EXPECT_NEAR(0.69, betNormalized.centralization(), 0.01);
+}
+
+TEST_F(CentralityGTest, testLocalPartitionCoverage) {
+    /* Graph:
+     0    3
+      \  / \
+       2    5
+      /  \ /
+     1    4
+    */
+    count n = 6;
+    Graph G(n);
+
+    G.addEdge(0, 2);
+    G.addEdge(1, 2);
+    G.addEdge(2, 3);
+    G.addEdge(2, 4);
+    G.addEdge(3, 5);
+    G.addEdge(4, 5);
+
+    ClusteringGenerator clustGen;
+    Partition P = clustGen.makeContinuousBalancedClustering(G, 3);
+
+    LocalPartitionCoverage LPC(G, P);
+    LPC.run();
+    EXPECT_EQ(1, LPC.maximum());
+    EXPECT_NEAR(LPC.centralization(), 0.29, 0.01);
+}
+
+TEST_F(CentralityGTest, testSfigality) {
+    /* Graph:
+     0    3
+      \  / \
+       2    5
+      /  \ /
+     1    4
+    */
+    count n = 6;
+    Graph G(n);
+
+    G.addEdge(0, 2);
+    G.addEdge(1, 2);
+    G.addEdge(2, 3);
+    G.addEdge(2, 4);
+    G.addEdge(3, 5);
+    G.addEdge(4, 5);
+
+    Sfigality sf(G);
```

Comment:
Technically, the following is out of the scope of this PR, however a fix should be trivial: Can you also change `Sfigality::maximum` (implementation + documentation)? Just return the maximum value of `scoreData` if `hasRun` is `true`. In addition, also add a test for that functionality.